src/design-system/components/icon-button/IconButton.tsx

```diff
@@ -56,11 +60,9 @@ export const IconButton: FC<S2SIconButtonProps> = ({
       onClick={onClick}
       {...props}
     >
-      {loading ? (
-        loader
-      ) : (
-        <AddRoundedIcon className={classNamesContainerIcon} />
-      )}
+      {loading
+        ? loader
+        : children || <AddRoundedIcon className={classNamesContainerIcon} />}
```

Comment:
Extract this to separate variable, please. Something like `buttonContent`

Reply:
And I you sure that AddRoundedIcon should be as default value here?

Reply:
> And I you sure that AddRoundedIcon should be as default value here?

Yes, AddRoundedIcon should be here by default, but since IconButton is a wrapper for other components, children ensures the visibility of those inner components. Without children, IconButton does not work properly.

Reply:
> Extract this to separate variable, please. Something like `buttonContent`

Done.